src/new-client/src/app/lib/components/nhs-header.tsx

```diff
@@ -1,31 +1,10 @@
-import { When } from '@components/when';
-import NhsHeaderLogIn from '@components/nhs-header-log-in';
-import NhsHeaderLogOut from '@components/nhs-header-log-out';
 import { Header } from '@nhsuk-frontend-components';
+import { ReactNode } from 'react';
 
 type NhsHeaderProps = {
-  userEmail?: string;
-  // We still show the header in the error handling page, which has no access to the userProfile call
-  // So to avoid it showing "Log in" when you're already logged in, we need to pass this prop for now
-  // This is a temporary solution until we have a better way to handle this
-  showAuthControls?: boolean;
+  children?: ReactNode;
 };
 
-export const NhsHeader = ({
-  userEmail,
-  showAuthControls = true,
-}: NhsHeaderProps) => {
-  return (
-    <Header>
-      <When condition={showAuthControls}>
-        <When condition={userEmail !== undefined}>
-          <span className="nhsuk-header-custom__user-control">{userEmail}</span>
-          <NhsHeaderLogOut />
-        </When>
-        <When condition={userEmail === undefined}>
-          <NhsHeaderLogIn />
-        </When>
-      </When>
-    </Header>
-  );
+export const NhsHeader = ({ children }: NhsHeaderProps) => {
```

Comment:
I'll play around with it some more - the split came when I was trying to use NextJS' interleaving (basically you can have a client component render a server component if the server component is passed in as a prop). This was when I was trying to make the error page (client) render the auth controls (server). I then just left the split in because it looked simpler to me. Will take another look at it